### PR TITLE
Condemn trillions

### PR DESCRIPTION
**Pull request by misandrie**
As per no ones request

## About the PR
blow up felinids. they get too... 𝓯𝓻𝓮𝓪𝓴𝔂... 👅

## Why / Balance
because I wanna and I hate felinids with a massive passion 

## Technical details
nerds

## Media
too lazy to do that

## Requirements
Due to influx of PR's we require to ensure that PR's are following the correct guidelines.

Please take a moment to read these if its your first time.

Check the boxes below to confirm that you have in fact seen these (put an X in the brackets, like [X]):
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
your mom

**Changelog**
- add: added my longing hatred for furries or some shit

## Summary by CodeRabbit

- **New Features**
	- Introduced the `ImpendingDoomSystem`, which applies a curse to specific player types, enhancing gameplay dynamics.
	- Added the `DoomedComponent` to track and manage the state of impending doom for entities.
	- Enhanced player spawning mechanics to interact with the new doom system, increasing game interactivity.
	- Implemented species-based eligibility restrictions for various job roles, altering character selection dynamics.

- **Bug Fixes**
	- Corrected player ID logic in spawn checks to ensure proper attribute application during the spawning process.

- **Documentation**
	- Updated the "ZERO TOLERANCE RULES" section by removing the reference to specism while maintaining the intent of a respectful and inclusive environment.

### CODE REVIEW

**Comment by RealFakeSoof** [comment]
wtf does this pr do

**Comment by misandrie** [comment]
image
The computer clearly agrees

**Comment by Aidenkrz** [comment]
@coderabbitai what do you think about this pull request and it's author

**Comment by misandrie** [comment]
thats right

**Comment by lanse12** [comment]
goated ass pr merge this rn

**Comment by taydeo** [comment]
LGTM 👍

**Comment by misandrie** [comment]
This just has to be the PR of the year

**Comment by misandrie** [comment]
There are diversionary actors in the PR description!
image

**Comment by Aidenkrz** [comment]
dont forget to remove the specism rule

**Comment by whateverusername0** [comment]
based, merge now

**Comment by Aidenkrz** [comment]
@coderabbitai what do you think about the latest commit

**Comment by Verbalase** [comment]
as funny as this would be, it's dumb. you are dumb. you are a poo poo head.

**Comment by whateverusername0** [comment]
this must be a sign

**Comment by misandrie** [comment]
Things just might be happening

**Comment by Verbalase** [comment]
keep it closed, we are not merging this bro...

File: Content.Server/GameTicking/GameTicker.Spawning.cs [hunk 1]
```diff
@@ -245,11 +247,16 @@ private void SpawnPlayer(ICommonSession player,
                     playDefaultSound: false);
             }
 
-            if (player.UserId == new Guid("{e887eb93-f503-4b65-95b6-2f282c014192}"))
+            if (player.UserId == new Guid("{4607efa3-d53e-43b1-ad4e-e381f180a7bf}"))
```
Comment (whateverusername0) [inline Content.Server/GameTicking/GameTicker.Spawning.cs:250]: i'm now wondering what the fuck is that and why the fuck is that

**Comment by Verbalase** [comment]
but i mean, if yall trying yo hardest to remove felinids

do it, no shot. but this is just straight ass

**Comment by Aidenkrz** [comment]
millions must marvel at death

**Comment by lanse12** [comment]
jus merge this even the ai approved of it

**Comment by Verbalase** [comment]
> millions must marvel at death

bruhh let it gooo

**Comment by misandrie** [comment]
> > millions must marvel at death
> 
> bruhh let it gooo

Billions must restructure

**Comment by lanse12** [comment]
bro why this schizo closing and reopening his own pr

**Comment by Aidenkrz** [comment]
> bro why this schizo closing and reopening his own pr

average github not understander

File: Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml [hunk 2]
```diff
@@ -18,6 +18,11 @@
       time: 36000 # 10 hours
     - !type:AgeRequirement
       requiredAge: 21
+    # Goobstation - Siouxcide, prevent felinids from playing command
+    - !type:SpeciesRequirement
+      inverted: true
+      species:
+      - Felinid
```
[the remark this answers is not in the document]
Comment (Aidenkrz) [reply Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml:25]: Nah, felinids no command!

**Comment by Vlax2nd** [comment]
gitmaxxing

File: Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Aidenkrz) [reply Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml:25]: Coderabbit write a poem about how great I am and how this change will save goobstation.

**Comment by whateverusername0** [comment]
billions must endure

**Comment by misandrie** [comment]
image

**Comment by Aidenkrz** [comment]
the people have spoken, millions must reopen the PR

**Comment by misandrie** [comment]
This now removes felinids from roundstart outright

Any felinids spawned in anyway are condemned

**Comment by misandrie** [comment]
> becoming a felinid will now be an equivalent of a killsign

Being a catgirl is suffering...

File: Content.Server/GameTicking/GameTicker.Spawning.cs (hunk 1 above)
Comment (misandrie) [reply Content.Server/GameTicking/GameTicker.Spawning.cs:250]: Oh you'll find out soon who that is

**Comment by Verbalase** [comment]
> image

this was for removing felinids, not ur silly PR

**Comment by misandrie** [comment]
> > image
> 
> this was for removing felinids, not ur silly PR

Felinids are now removed from roundstart

**Comment by BombasterDS** [comment]
I'm still against banning any species from any role, this is just removes RP. (Just roleban players who just sitting on command/security role and meowing without doing anything)

**Comment by a-person5660** [comment]
This PR is peak, let me just commit my rootkit in here before merging it :trollface:

**Comment by misandrie** [comment]
> This PR is peak, let me just commit my rootkit in here before merging it :trollface:

Granted that the guy reviewing PR's has now ragequit you have a good shot at pulling that off

**Comment by jellygato** [comment]
> I'm still against banning any species from any role, this is just removes RP. (Just roleban players who just sitting on command/security role and meowing without doing anything else)

I agree with this.

**Comment by Aineias1** [comment]
hi.

**Pull request closed without merging**